### ISSUE TITLE
Units of mRNA half life

### ISSUE THREAD

**Issue by shtoneyan** [vagarwal87/saluki_paper#12]
I was wondering if the final processed dataset labels (representing mRNA half life) are derived from the PC1 from PCA of all the datasets. If not, what is the unit of the final training dataset labels? Thanks!

**Comment by davek44** [comment]
Hi, yes that is basically correct, with the only caveat that we removed some outlier datasets before the final PCA. This is described in the [Methods of the paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02811-x#Sec11).
